index.html

```diff
@@ -67,7 +67,7 @@ <h3 class="year">2015</h3>
                 </div>
             </div>
             <div class="container-reverse">
-                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-2.png" alt="snapshout portfolio">
+                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-2.png" alt="Screenshot of the Multi-Post Stories project">
```

Comment:
Changed and added more detailed descriptions of the images in the alternative attributes.